test/torch_np/test_basic.py

```diff
@@ -487,7 +489,7 @@ def test_divmod_out_list(self):
         assert quot is out[0]
         assert rem is out[1]
 
-    @pytest.mark.xfail(reason="out1, out2 not implemented")
+    @xfail  # ("out1, out2 not implemented")
```

Comment:
why not keeping the comment?

Reply:
It just does not seem to have a relevant argument.

```
In [1]: from unittest import expectedFailure

In [2]: expectedFailure?
Signature: expectedFailure(test_item)
Docstring: <no docstring>
File:      ~/.conda/envs/pytorch-dev/lib/python3.8/unittest/case.py
Type:      function
```